src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -113,6 +116,11 @@ function NameAndPicture({ navigation, route }: Props) {
       return
     }
 
+    try {
+      Statsig.logEvent(StatsigEvents.ONBOARDING_NAME_STEP_COMPLETE)
```

Comment:
nit: since the other statsig stuff for this component is taken out of this PR, might make sense to separate this out as well

Reply:
to keep it single purpose